factory/usr/lib/systemd/system/systemd-bootchart.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Boot Process Profiler
+Documentation=man:systemd-bootchart.service(1) man:bootchart.conf(5)
+DefaultDependencies=no
+
+[Service]
+ExecStartPre=/usr/sbin/mkdir -p /run/log
```

Comment:
nitpick, but instead of doing this, I would rather see the config file change the output to somewhere that we don't need to mkdir -p via `Output=` but either way I suppose

Reply:
Not much is in `/run` when the initrd is running, so there are not that many options... we could directly store the chart there, but I think it is nice to have things in a subfolder so things are ordered. If you feel we really should avoid the mkdir, I would put `Output=/run` in config.

Reply:
/run/log is fine I suppose in that case